From 3fa2f40f88ac0e19b46743552c9bf7bd87aaf0c6 Mon Sep 17 00:00:00 2001
From: Edward Toroshchin <kde@hades.name>
Date: Sat, 2 Feb 2013 19:45:51 +0100
Subject: [PATCH 02/10] fileindexer: fix taglib bitrate units

taglib uses kbit/s for the bitrate unit, whereas nfo:averageBitrate is
in bit/s.

REVIEW: 108723
---
 services/fileindexer/indexer/taglibextractor.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/fileindexer/indexer/taglibextractor.cpp b/services/fileindexer/indexer/taglibextractor.cpp
index e6ba162..932c256 100644
--- a/services/fileindexer/indexer/taglibextractor.cpp
+++ b/services/fileindexer/indexer/taglibextractor.cpp
@@ -132,7 +132,7 @@ Nepomuk2::SimpleResourceGraph TagLibExtractor::extract(const QUrl& resUri, const
         }
 
         if( audioProp->bitrate() ) {
-            fileRes.setProperty( NFO::averageBitrate(), audioProp->bitrate() );
+            fileRes.setProperty( NFO::averageBitrate(), audioProp->bitrate() * 1000 );
         }
 
         if( audioProp->channels() ) {
-- 
1.8.1.2

